src/EnergyPlus/DataDaylighting.hh

```diff
@@ -95,88 +96,89 @@ namespace DataDaylighting {
         DElightDaylighting
     };
 
-    // Parameters for "Lighting Control Type"
-    enum class iLtgCtrlType
+    // Parameters for "Lighting Control Type" - these are the values expected by DElight
+    enum class LtgCtrlType
     {
+        Invalid = 0,
         Continuous = 1,
         Stepped = 2,
-        ContinuousOff = 3
+        ContinuousOff = 3,
```

Comment:
This does compile (why?) but leaves me waiting for the next type (anytime I see a comma).